timeline.js

```diff
@@ -1,43 +1,43 @@
-import { ActivityPubDB } from "./db.js";
+import { db } from "./dbInstance.js";
 
 class ReaderTimeline extends HTMLElement {
   constructor() {
     super();
-    // Default outbox URLs to fetch and display in the timeline
-    this.outboxUrls = [
-      "ipns://staticpub.mauve.moe/outbox.jsonld",
-      "ipns://hypha.coop/outbox.jsonld",
+    this.actorUrls = [
+      "https://staticpub.mauve.moe/about.jsonld",
+      "https://hypha.coop/about.jsonld",
+      "https://prueba-cola-de-moderacion-2.sutty.nl/about.jsonld",
     ];
+    this.processedNotes = new Set(); // To keep track of notes already processed
   }
 
   connectedCallback() {
     this.initTimeline();
   }
 
   async initTimeline() {
-    const db = await ActivityPubDB.load();
+    this.innerHTML = ""; // Clear existing content
 
-    // Clear existing content
-    this.innerHTML = "";
-
-    for (const outboxUrl of this.outboxUrls) {
+    for (const actorUrl of this.actorUrls) {
       try {
-        console.log("Ingesting actor from outbox URL:", outboxUrl);
-        await db.ingestActor(outboxUrl);
-
-        const outboxElement = document.createElement("distributed-outbox");
-        outboxElement.setAttribute("url", outboxUrl);
-
-        this.appendChild(outboxElement);
+        console.log("Loading actor:", actorUrl);
+        const actorData = await db.ingestActor(actorUrl);
+        const notes = await db.searchNotes({ attributedTo: actorData });
```

Comment:
Instead, ingest all the actors and search notes once at the end without filtering by attributedTo